bors.toml

```diff
@@ -1,7 +1,16 @@
 status = [
   "formatting",
-  "clippy",
   "lint-commits",
+  "clippy (daemon)",
+  "clippy (rocket-basicauth)",
+  "clippy (rust-embed-rocket)",
+  "clippy (taker)",
+  "clippy (maker)",
+  "clippy (shared-bin)",
+  "clippy (daemon-tests)",
+  "clippy (tokio-tasks)",
+  "clippy (xtras)",
+  "block-fixup",
```

Comment:
Must have been a rebase errr, this check no longer exists!

That is why bors is timing out.

Reply:
Thanks for the fix!